node/lib/cmd/open.js

```diff
@@ -118,6 +118,11 @@ exports.executeableSubcommand = co.wrap(function *(args) {
             return;                                                   // RETURN
         }
 
+        if (shas[index] === null) {
```

Comment:
can we rename index here? Two index variables in the same function is really confusing.

Reply:
Doing that in a separate commit.